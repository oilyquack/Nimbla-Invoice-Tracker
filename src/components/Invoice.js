import React from "react";

function Invoice({ invoiceId, amount, payBy, customer, services }) {
  return (
    <div>
      <h2>{invoiceId}</h2>
      <p>{amount}</p>
      <p>{payBy}</p>
      <p>{customer}</p>
      <p>{services}</p>
    </div>
  );
}

export default Invoice;
