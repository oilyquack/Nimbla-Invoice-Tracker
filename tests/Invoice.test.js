import React from "react";
import renderer from "react-test-renderer";
import Invoice from "../src/components/Invoice";

it("matches the snapshot", () => {
  const tree = renderer.create(<Invoice />);
  expect(tree).toMatchSnapshot();
});
