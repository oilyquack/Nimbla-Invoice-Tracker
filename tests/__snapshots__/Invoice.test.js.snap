// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="app__invoices__display--read"
>
  <h2>
    Invoice # 
  </h2>
  <p>
    Amount: Â£
  </p>
  <p>
    Pay By: 
  </p>
  <p>
    Customer: 
  </p>
  <p>
    Services: 
  </p>
  <p>
    Unpaid
  </p>
  <button
    onClick={[Function]}
  >
    Edit
  </button>
</div>
`;
